docs/sovereign/from-sovereign.md

```diff
@@ -0,0 +1,158 @@
+# Execution starting from a Sovereign Chain
+![From Sovereign](../../static/sovereign/from-sovereign.png)
+
```

Comment:
First specify all the capabilities:
The solution enables users will be able to *transfer tokens* and *execute SC* from sovereign to main chain.
Then you start to describe each case (transfer, execute) below